backend/api/tests/handlers/dynamodb/put-service-client.test.ts

```diff
@@ -85,8 +100,9 @@ describe("putServiceClientHandler tests", () => {
         const error = "SomeAwsError";
         const putItemSpy = jest.spyOn(DynamoDbClient.prototype, "put").mockRejectedValue(error);
 
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
```

Comment:
Same as above

Reply:
Thanks. Yes: this makes sense and I've worked through cleaning these up.